concourse/tasks/aiven-broker-acceptance-tests-run.yml

```diff
@@ -4,7 +4,7 @@ image_resource:
   type: docker-image
   source:
     repository: ghcr.io/alphagov/paas/cf-acceptance-tests
-    tag: 168a6c8b43fdec99607fd8bc01c5986a46a72dc0
+    tag: 7191ee87b98642e5d03a80f24585fddc02fc8d88
```

Comment:
Could we merge the changes in this commit in to the first commit that changes the hashes?